model/model.go

```diff
@@ -206,6 +208,89 @@ func (model Model) PrintModel() {
 	model.GetLogger().LogModel(modelInfo)
 }
 
+func (model Model) SortPoliciesBySubjectHierarchy() error {
+	if model["e"]["e"].Value != "subjectPriority(p_eft) || deny" {
+		return nil
+	}
+	subIndex := -1
+	for ptype, assertion := range model["p"] {
+		for index, token := range assertion.Tokens {
+			if token == fmt.Sprintf("%s_sub", ptype) {
+				subIndex = index
+				break
+			}
+		}
+		if subIndex == -1 {
+			continue
+		}
```

Comment:
Can we assume `sub` always to be first, like `SortPoliciesByPriority`

Reply:
In ``SortPoliciesByPriority``,I think it does’t assume that ``sub`` always to be first.
``sub`` and ``domain`` ‘s position are not fixed.Do we need to fix the position of ``sub``and ``dom`` in the model like ``g``?It seems simpler